### DISCUSSION TITLE
@babel/core npm version

### DISCUSSION THREAD

**Discussion by GiovaniBiagi** [babel/babel#14577] [General] [answered]
Why is the latest version of the lib not available for download in npm? I'm trying to fix the problem in the image bellow, but in npm we don't have the latest (and fixed) version for download:

the problem:
![image](https://user-images.githubusercontent.com/49461601/169899224-83d391e1-3019-4a16-b380-d62003306a5d.png)

the npm version:
![image](https://user-images.githubusercontent.com/49461601/169893675-070dba35-3a33-413a-baf6-2da402e0bb00.png)

**Comment by nicolo-ribaudo** [comment, marked as answer]
That has been fixed in v7.18.1 of `@babel/plugin-transform-typescript`, sorry for the confusion!

**Reply by GiovaniBiagi** [reply]
ahh now i understand haha ty so much!!
